docs/conf.py

```diff
@@ -23,7 +21,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "RDK"
-copyright = "2017-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved"
+copyright = "2017-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved"
```

Comment:
We likely shouldn't be maintaining a range, but rather the original year of copyright per OSS guidance.  That said, since we're maintaining a range already, updating it to show we're still maintaining the project :D